src/backends/sentry_backend_crashpad.cpp

```diff
@@ -451,6 +468,7 @@ sentry__backend_new(void)
     backend->user_consent_changed_func
         = sentry__crashpad_backend_user_consent_changed;
     backend->get_last_crash_func = sentry__crashpad_backend_last_crash;
+    backend->prune_database_func = sentry__crashpad_backend_prune_database;
```

Comment:
are all other backend guaranteed to initialise this to NULL?

Reply:
yes, we are `memset`-ing the backend struct to 0